app/views/items/edit.html.erb

```diff
@@ -1,20 +1,22 @@
-<% content_for(:title, "Edit Item") %>
-<h1>Edit Item</h1>
-<%= simple_form_for @item, url: item_path(@item), method: :patch do |f| %>
-  <%= f.error_notification %>
+<div class="container align-justify-center">
+  <% content_for(:title, "Edit Item") %>
+  <h1>Edit Item</h1>
+  <%= simple_form_for @item, url: item_path do |f| %>
+    <%= f.error_notification %>
 
-  <%= f.input :title, label: 'Iphone:' %>
+    <%= f.input :title, placeholder: 'Macbook', label:'Give a sexy Title:' %>
 
-  <%= f.input :description, label: 'Useful for Students:' %>
+    <%= f.input :description, placeholder: 'Great for student', label:'Description:' %>
 
-  <%= f.input :condition, label: 'Excellent:' %>
+    <%= f.input :condition, placeholder: 'Excellent', :collection => %w[Excellent Good Average Bad Broken] %>
 
-  <%= f.input :year_made, label: '2020:' %>
+    <%= f.input :year_made, placeholder: '2020', label:'Year Made:', min_max: true %>
 
-  <%= f.input :number_available, label: '1:' %>
+    <%= f.input :number_available, placeholder: '1', label:'Number Available:' %>
```

Comment:
`min_max: true` for number available too

Reply:
Done